test/test_ad.jl

```diff
@@ -124,9 +124,11 @@ forwarddiff_color_jacobian(_oop_jacout, oopf, x; colorvec = repeat(1:3,10), spar
 @test typeof(_oop_jacout) == typeof(_J)
 @test fcalls == 1
 
-_oop_jacout = BandedMatrix(1.01 .* _oop_jacout) # check w/BandedMatrix instead of sparse
+# BandedMatrix
+_oop_jacout = BandedMatrix(-1 => diag(J, -1) .* 1.01, 0 => diag(J, 0) .* 1.01,
+                           1 => diag(J, 1) .* 1.01) # check w/BandedMatrix instead of sparse
 fcalls = 0
-forwarddiff_color_jacobian(_oop_jacout, oopf, x; colorvec = repeat(1:3,10), sparsity = _J, jac_prototype = _J)
+forwarddiff_color_jacobian(_oop_jacout, oopf, x; colorvec = repeat(1:3,10), sparsity = _J)
 @test _oop_jacout ≈ J
 @test isa(_oop_jacout, BandedMatrix)
 @test fcalls == 1
```

Comment:
A tridiagonal matrix should have 3 f calls? Especially with that colorvec.

Reply:
What about the test at line 114-117? I didn't write that test, but it is essentially the same test I am running (in fact, that's what I based my original test off of).

Reply:
oh wait, I guess it is one f call because of partial chunking. My bad.